commands/command_track.go

```diff
@@ -87,11 +90,13 @@ ArgsLoop:
 		// Since all `git-lfs track` calls are relative to the root of
 		// the repository, the leading slash is simply removed for its
 		// implicit counterpart.
+		Print("Searching for files matching pattern: %s", pattern)
 		gittracked, err := git.GetTrackedFiles(pattern)
 		if err != nil {
 			LoggedError(err, "Error getting git tracked files")
 			continue
 		}
+		Print("Found %d files matching pattern: %s", len(gittracked), pattern)
```

Comment:
I'd only print this in verbose mode, and I'd make it more specific that it's talking about files *already added* to git. Otherwise it would sound weird if you hadn't added any files to git yet but had files on disk with that pattern and it said "Found 0 files".